src/dmd/globals.d

```diff
@@ -118,6 +118,14 @@ enum CxxHeaderMode : uint
     verbose /// Generate headers and add comments for hidden declarations
 }
 
+/// Trivalent boolean to represent the state of a `revert`able change
+enum RevertState
```

Comment:
Revert is a rather confusing name as RevertState.enabled means now sth very confusing.
How about `FeatureState`?

Reply:
Even `PreviewState` would be a less confusing name...

Reply:
> Revert is a rather confusing name as RevertState.enabled means now sth very confusing.
> How about `FeatureState`?

That sounds better. Fixed